### ISSUE TITLE
PMID object in "ValidContradictoryEvidence"

### ISSUE THREAD

**Issue by jimmyzhen** [clingen-data-model/data-exchange-shared-json#19]
```
"ValidContradictoryEvidence": 
{
	"YesNo": "YES",
	"pmid": 
	{
		"publications": [
			{
				"title": "[Oxygen affinity of haemoglobin (author",
				"author": "Robert M",
				"pubdate": "1975 Jan-Feb",
				"source": "Bull Physiopathol (Nancy)",
				"uid": "111"
			}
		],
		"notes": 
		{
			"note": "Proband with "
		}
	}
},
```

Hi @sgoehringer,

Is there a reason for including the PMID in this object?

**Comment by sgoehringer** [comment]
@jimmyzhen This was related to this ticket https://github.com/clingen-data-model/data-exchange-shared-json/issues/14#issuecomment-333619809

"What do you think about 'evidence'?"

**Comment by jimmyzhen** [comment]
Thanks.

**Comment by sgoehringer** [comment]
Addressed - 8cf6e4b5ba67c0893a970c11b5628a974078a27a
